README.md

```diff
@@ -1,6 +1,5 @@
 <div align="center">
 
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/218262265-8fc0efd4-742e-46f2-9743-688798c58842.png)

- [ ] Kindly put some future features if cannot say specifically what feature would be announced then at least tell use about it generally in this section or talk about any problem for which you are going to find a solution :+1: